doc/users/github_stats.rst

```diff
@@ -1,9 +1,9 @@
 .. _github-stats:
 
-GitHub Stats
-============
+GitHub statistics
+=================
 
-GitHub stats for 2021/03/31 - 2021/05/07 (tag: v3.4.1)
+GitHub statistics for 2021/03/31 - 2021/05/07 (tag: v3.4.1)
```

Comment:
This file is autogenerated, and you haven't changed the generator.

Reply:
Changed the generator as well.